web_src/js/components/DashboardRepoList.vue

```diff
@@ -355,9 +355,9 @@ export default sfc; // activate the IDE's Vue plugin
         </a>
       </h4>
       <div class="ui attached segment repos-search">
-        <div class="ui small fluid action left icon input" :class="{loading: isLoading}">
+        <div class="ui small fluid action left icon input">
           <input type="search" spellcheck="false" maxlength="255" @input="changeReposFilter(reposFilter)" v-model="searchQuery" ref="search" @keydown="reposFilterKeyControl" :placeholder="textSearchRepos">
-          <i class="icon"><svg-icon name="octicon-search" :size="16"/></i>
+          <i class="icon" :class="{'is-loading': isLoading}"><svg-icon name="octicon-search" :size="16"/></i>
```

Comment:
It could also add a `small-loading-icon` to make the icon thinner.

Reply:
Ok, looks almost too thin, but ok.

<img width="174" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/165d791b-1618-4efe-83ee-58676c020193">

Reply:
Okay I renamed this class to `is-loading-2px` and added a `is-loading-3px` used here:

<img width="204" alt="Screenshot 2024-03-31 at 14 39 33" src="https://github.com/go-gitea/gitea/assets/115237/73ecfaa4-a358-4149-9bb2-d89dc2133ae1">

Reply:
@wxiaoguang looking good?

Reply:
TBH it looks strange. Just like writing something `tw-border tw-border-1`, it is counter-intuitive and seems wordy. Personally I prefer a clear name: `class="is-loading loading-icon-small` (loading-icon-medium)

Reply:
I think it's better to prefix with the loading class name as it's only meant to be used together.

Reply:
Renamed.